elements/pfe-navigation/src/_variables-mixins.scss

```diff
@@ -0,0 +1,254 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: navigation;
+
+$menu-height: 72px;
+
+$gutter: 32px;
+
+$breakpoints: (
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1200px,
+  2xl: 1368px,
+  // The below should match an above breakpoint value
+  // Where the icons on the right appear
+  'secondary-links-expand': 768px,
+
+  // Default value for when the nav expands to desktop mode
+  'nav-expand': 1200px,
+);
+
+/// Breakpoint helper specific to pfe-navigation
+/// @param {string} $breakpoint Name of a breakpoint
+/// @see $breakpoints
+@mixin breakpoint($breakpoint) {
```

Comment:
The existing breakpoint function allows for custom breakpoints to be used.  They can be added to the $LOCAL-VARIABLES map and the function will automatically look-up for you by using the `$map: false` flag.

```
Set $map to false if you need supplemental breakpoints in a component  
/// Use:
/// ```
/// $LOCAL-VARIABLES: (
///    lg-mobile:         640,
///    sm-desktop:        1024,
/// );
/// ```
///
/// ```
/// @media screen and (min-width: pfe-breakpoint(lg-mobile, $map: false)) { }
/// ```
/// which returns:
/// ```
/// @media screen and (min-width: var(--pfe-navigation--breakpoint-lg-mobile, 640)) { }
/// ```
```